horizon/files/apache_extra_opts.conf

```diff
@@ -0,0 +1,5 @@
+{%- from "horizon/map.jinja" import server with context %}
+
+{%- for apache_extra_opts in server.apache_extra_opts %}
+  {{ apache_extra_opts }}
```

Comment:
Defining string in metadata and just passing it into config is not correct approach. Take inspiration from apache formula and design proper metadata structure for use cases you are trying to satisfy.
Alternatively try to elaborate on how to use apache formula instead.